ui/src/App.tsx

```diff
@@ -14,12 +14,13 @@ import {
 } from '@agoric/web-components';
 import { subscribeLatest } from '@agoric/notifier';
 import { stringifyAmountValue } from '@agoric/ui-components';
-import { makeCopyBag } from '@agoric/store';
 
 type Wallet = Awaited<ReturnType<typeof makeAgoricWalletConnection>>;
 
 export const contractName = 'swaparoo';
-const gov1 = 'agoric14t2eagaphnkj33l9hvcrf90t3xffkt0u3xy6uh';
+// const gov1 = 'agoric14t2eagaphnkj33l9hvcrf90t3xffkt0u3xy6uh';
+const swappa = 'agoric13whlmf7akvmg05n3zx37v7r0htyasa4m28j9cu';
```

Comment:
Can we just do `const recipientAddr = agoric13whlmf7akvmg05n3zx37v7r0htyasa4m28j9cu`? This variable name is a bit cryptic and seems redundant